cmd/cni-metrics-helper/main.go

```diff
@@ -88,7 +99,7 @@ func main() {
 	// should be name/identifier for the cluster if specified
 	clusterID, _ := os.LookupEnv("AWS_CLUSTER_ID")
 
-	log.Infof("Starting CNIMetricsHelper. Sending metrics to CloudWatch: %v, LogLevel %s", options.submitCW, logConfig.LogLevel)
+	log.Infof("Starting CNIMetricsHelper. Sending metrics to CloudWatch: %v, LogLevel %s, pullInterval %d", options.submitCW, logConfig.LogLevel, pullInterval)
```

Comment:
looks like this needs to be updated to new var name: `metricUpdateInterval`

Reply:
My apologies. Updated.